app/webroot/js/wjr/map.js

```diff
@@ -75,13 +75,15 @@ define(['jquery', 'underscore', 'async!https://maps.googleapis.com/maps/api/js?k
 
   map.MultiMarkerMap = function () {
     this.initialize = function (element) {
-      var lat, fetchUrl, fetchEntity, fetchMarkers, fetchBounds,
-        lng, zoom, googleMap, options, center, markers, fetchInProgress;
+      var lat, fetchDomain, fetchUrl, fetchEntity, fetchMarkers, fetchMarkerImages, fetchBounds,
```

Comment:
no idea why i did it this way that we declare all the variables before assigning them. Not related to your PR anyways :)